app/components/support_interface/provider_courses_table_component.rb

```diff
@@ -34,15 +34,16 @@ def accredited_bodies_vary?
     attr_reader :provider, :courses
 
     def status_tag(course)
-      # Special case when the course is in the future but has been imported with a status of open_on_apply
-      return govuk_tag(text: 'Closed on Apply', colour: 'blue') if course.recruitment_cycle_year == RecruitmentCycle.next_year
-
-      if course.open_on_apply?
-        govuk_tag(text: open_on_apply_message(course), colour: 'green')
-      elsif course.exposed_in_find?
-        govuk_tag(text: 'Closed on Apply', colour: 'blue')
-      else
+      if course.recruitment_cycle_year == RecruitmentCycle.next_year
+        govuk_tag(text: 'Unpublished', colour: 'blue')
+      elsif course.open?
+        govuk_tag(text: 'Open', colour: 'green')
+      elsif !course.exposed_in_find?
         govuk_tag(text: 'Hidden in Find', colour: 'grey')
+      elsif course.application_status_closed?
+        govuk_tag(text: 'Closed by Provider', colour: 'blue')
```

Comment:
I think this should be 'Closed by provider' because we tend to use informal capitalisation and we don't appear to treat 'provider' as a proper noun in other places. (unlike Find or Apply)